templates/vite-cloudflare/.gitignore

```diff
@@ -3,5 +3,6 @@ node_modules
 /.cache
 /build
 .env
+.dev.vars
```

Comment:
I think you could also make the same change for the `cloudflare-pages` and `cloudflare-workers` templates.

Reply:
Those templates won't be maintained anymore

Reply:
There is now a [cloudflare-workers](https://github.com/remix-run/remix/tree/bcde53b2e6258756e5053129a84ef3f929e7fb31/templates/cloudflare-workers) for vite

I'll go ahead and update it though in a separate PR, because I have some other updates to make